types/src/byte_code.rs

```diff
@@ -30,6 +30,9 @@ const BYTE_CODE_PREFIX: &str = "byte-code-";
 const V1_WASM_PREFIX: &str = "v1-wasm-";
 const EMPTY_PREFIX: &str = "empty-";
 
+/// A special contract wasm hash for contracts representing Accounts.
+pub const ACCOUNT_BYTE_CODE_HASH: ByteCodeHash = ByteCodeHash::new([0; 32]);
```

Comment:
It seems like this is not used?

Reply:
Removed in [d8cbbdf](https://github.com/casper-network/casper-node/pull/4544/commits/d8cbbdfe05181fbe83a5f667364f18bd2a8ab052).